src/components/screens/wallet/delegatePerformance/index.js

```diff
@@ -0,0 +1,36 @@
+/* istanbul ignore file */
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { getActiveTokenAccount } from '@utils/account';
+import { getDelegate } from '@api/delegate';
+import withData from '@utils/withData';
+import { parseSearchParams } from '@utils/searchParams';
+import { withTranslation } from 'react-i18next';
+import DelegatePerformance from './delegatePerformance';
+
+const mapStateToProps = (state, ownProps) => ({
+  account: getActiveTokenAccount(state),
+  id: ownProps.match.params.id,
+  activeToken: state.settings.token?.active ?? 'LSK',
```

Comment:
Why do you need account, id and activeToken here?

Reply:
It was unintentional. Removed